tools/benchlocks/benchlocks.go

```diff
@@ -0,0 +1,73 @@
+package benchlocks
```

Comment:
A bit of package documentation is likely warranted.  Can be a 1-liner. https://go.dev/blog/godoc